examples/entity-viewer-v2/entity-viewer-client/src/main/ml-modules/root/explore-data/ui-config/config.json

```diff
@@ -258,6 +258,7 @@
           "arrayPath": "extracted.person.nameGroup",
           "path": "fullname.value"
         },
+        "paginationSizes": [3,5,10,20],
```

Comment:
Can we call this property "pageLengths" to be consistent with the "pageLength" value that it eventually determines?